runtime-config-linux.md

```diff
@@ -144,6 +144,12 @@ Also known as cgroups, they are used to restrict resource usage for a container
 cgroups provide controls to restrict cpu, memory, IO, pids and network for the container.
 For more information, see the [kernel cgroups documentation](https://www.kernel.org/doc/Documentation/cgroups/cgroups.txt).
 
+There are a few facets to this:
+
+* Managing the existence of cgroups within the hierarchy (`cgroupsPath`).
```

Comment:
On Wed, Dec 16, 2015 at 11:02:59AM -0800, Jonathan Boulle wrote:
> > +* Managing the existence of cgroups within the hierarchy (`cgroupsPath`).
> 
> I assume he means the unified hierarchy, since subsequently it
> mentions the spec does not support split, but would be good to make
> that explicit

I can make “the hierarchy” a link to [1].  But as far as I know, we
don't actually require you to use *that* unified-hierarchy
implementation.  We just don't allow you to give per-controller paths.
I think that's too complicated an idea for this summary-list, but I'll
happily add more detail to the existing split-hierarchy reference to
clarify this point.

[1]: https://www.kernel.org/doc/Documentation/cgroups/unified-hierarchy.txt